Test/PublishNodeModule.Tests.ps1

```diff
@@ -229,8 +272,8 @@ Describe 'PublishNodeModule.when publishing node module' {
         }
 "@
         WhenPublishingNodeModule -WithCredentialID 'NpmCred' `
-                                    -WithEmailAddress 'somebody@example.com' `
-                                    -WithNpmRegistryUri 'http://registry@example.com'
+                                 -WithEmailAddress 'somebody@example.com' `
+                                 -WithNpmRegistryUri 'http://registry@example.com'
         ThenNpmrcCreated -WithEmail 'somebody@example.com' -WithRegistry 'http://registry@example.com'
         ThenNpmPackagesPruned
         ThenNodeModulePublished

```

Comment:
Add assertion that we aren't passing any tags to `Invoke-WhiskeyNpmCommand`. Add to whatever tests it makes sense. Or add a `WithNoTag` switch to `ThenModeModulePublished`?